### ISSUE TITLE
Upload cover picture does not work when using subfolder

### ISSUE THREAD

**Issue by pitaj** [NodeBB/NodeBB#5631]
Uploaded cover picture path does not contain the `relative_path`.

**Comment by margussipria** [comment]
Please include the following information when submitting a bug report/issue:

* NodeBB version using tags `1.4.5` (d19e59f1e13f4a22eb44d5334b2a81d99e030b36) and `1.4.6` (fb2f857af2be85971f9a2d83c5458176b174c604)
* Database mongo, version 3.4.3
* Exact steps to cause this issue
    1. Upgraded version `1.0.3` -> `1.1.2` -> `1.2.1` -> `1.3.0` -> `1.4.5` (I haven't tried clean installation)
    2. Using url with `relative_path` (example. `/forum`)
    3. Uploading cover picture
* What you expected
    * I expect picture be loaded from `/forum/assets/uploads/profile/1-profilecover.jpeg`
* What happened instead
    * Server tries to load new pictures from `/assets/uploads/profile/1-profilecover.jpeg`

Thank you!

**Comment by barisusakli** [comment]
@margussipria what is the value saved in the database?

**Comment by margussipria** [comment]
Now I have stated with new installation with `config.json`:
```json
  {
    "url": "http://localhost:4567/bug",
    "secret": "bug",
    "database": "mongo",
    "port": "4567",
    "mongo": {
      "host": "mongo",
      "port": "27017",
      "username": "bug",
      "password": "bug",
      "database": "bug"
    }
  }
```

And only thing after login with new admin account I uploaded cover picture, that had same problem.

**Comment by pitaj** [comment]
This is confirmed. Working on a fix.

**Comment by margussipria** [comment]
Old uploads have `"cover:url" : "/forum/uploads/profile/2-profilecover"` and new uploads do not have it `"cover:url" : "/assets/uploads/profile/1-profilecover.jpeg"`. 

Old profile pictures have that prefix (`"picture" : "/forum/uploads/profile/2-profileimg.jpg"`)... and they are working... new uploads (`"picture" : "/assets/uploads/profile/1-profileavatar.jpeg"`) do now have it and they are also working.

My question is why this forum does not use 
```html
<head>
  <base href="http://localhost:4567/forum/" >
</head>
```

Then there is no need to prefix every resource, javascript and css, also it would be easier to include pictures from from css.

**Comment by pitaj** [comment]
#5632 will fix this. The fix should ship in v1.5.0
